docs/tutorial/Pre-requisites of running a Waku node.md

```diff
@@ -0,0 +1,116 @@
+---
+title: 'Pre-requisites of running a Waku node'
+---
+
+Pre-requisites of running a Waku node
+===
+
+## Table of Contents
+
+[TOC]
+
+## Creating Goerli Ethereum Account and obtaining its private key
+
+
+1. Download and install Metamask [https://metamask.io/download/](https://metamask.io/download/)
+2. Create a new wallet and save your secret recovery key
+    
+    ![](https://i.imgur.com/HEOI0kp.jpg)
+
+3. Login to Metamask
+    
+    ![](https://i.imgur.com/zFduIV8.jpg)
+
+4. You should already see an account created. As you can see on top right, it should be pointing to Ethereum mainnet.
+    
+    ![](https://i.imgur.com/gk3TWUd.jpg)
+
+5. You can use the same account for different networks.
+6. You can switch to a test network by clicking on the drop down menu like so:
+
+    ![](https://i.imgur.com/kbuup6y.jpg)
+
+7. Click on Show/hide test networks
+8. Enable “Show Test Networks
+    
+    ![](https://i.imgur.com/ztiyKsB.jpg)
+    
+    ![](https://i.imgur.com/UFsoRHR.jpg)
+
+9. Close the settings and now you should see the test networks in the drop down on the top right
+    
+    ![](https://i.imgur.com/leSYt4z.jpg)
+
+10. Switch to Goerli Test Network
+11. You can see that the same account can be used with different networks. But note that the ETH balance on different networks are different and are different tokens.
+    
+    ![](https://i.imgur.com/Y7YkuEe.jpg)
+
+12. Now to export your private key for the account, if needed, please click on the three dots beside the account.
+    
+    ![](https://i.imgur.com/GjwoDPE.jpg)
+
+13. Click on Account Details
+14. Click on Export Private Key
+    
+    ![](https://i.imgur.com/7g6SyuX.jpg)
+
+15. Enter your Metamask password when prompted
+    
+    ![](https://i.imgur.com/VOojHY5.jpg)
+
+16. You will be shown the private key. Copy it as needed.
+
+Obtain Goerli Ethers from faucet
+---
+
+1. Ethers on Goerli test networks can be obtained from different faucets
+2. Some of the faucets are as follows
+    1. [https://goerlifaucet.com/](https://goerlifaucet.com/)
+    2. [https://goerli-faucet.slock.it/](https://goerli-faucet.slock.it/)
+    3. [https://goerli-faucet.pk910.de/](https://goerli-faucet.pk910.de/)
+3. Please follow the instructions on the webpages of these faucets
+4. A lot of faucets limit the Goerli ETH to 0.05 ETH/day
+5. To obtain more eth, you can do some POW mining. One of those POW faucet is
+     [https://goerli-faucet.pk910.de/](https://goerli-faucet.pk910.de/)
+6. Enter your Eth account address, accept Captcha and start mining
```

Comment:
Add something like ' below we show how to obtain ether using one of the PoW faucet, but feel free to use another method`

Reply:
Also, please hide your own confidential info from the images e.g., your eth private key, the infura node address etc?